experiments/Atmos_LES/dycoms.jl

```diff
@@ -15,6 +16,9 @@ using CLIMA.MoistThermodynamics
 using CLIMA.PlanetParameters
 using CLIMA.VariableTemplates
 
+const clima_dir = dirname(CLIMA)
+include(joinpath(clima_dir, "..", "Parameters", "Parameters.jl"))
```

Comment:
At the moment we can just specify the relative path to the current file:
```suggestion
include(joinpath("..", "..", "Parameters", "Parameters.jl"))
```
Once we build the infrastructure to do it from toml files, we can add that as an option to the driver.

Reply:
Going to stick with this to avoid a variable number of ".."'s for each driver that needs this.